python/python/Ice/Util.py

```diff
@@ -1,51 +1,71 @@
 # Copyright (c) ZeroC, Inc.
 
+
+import asyncio
 import os
 import sys
 import IcePy
+from .InitializationData import InitializationData
 from .Properties import Properties
 from .Communicator import Communicator
-from .InitializationData import InitializationData
+from .LocalExceptions import InitializationException
+from .asyncio.EventLoopAdapter import EventLoopAdapter
 
 __name__ = "Ice"
 
 
-def initialize(args=None, data=None):
+def initialize(args=None, initData=None, configFile=None, eventLoop=None):
     """
-    Initialize a new Ice communicator.
-
-    This method initializes a new Ice communicator. The optional arguments represent
-    an argument list (such as sys.argv) and an instance of InitializationData.
-    You can invoke this function as follows:
-
-    - `Ice.initialize()`
-    - `Ice.initialize(args)`
-    - `Ice.initialize(data)`
-    - `Ice.initialize(args, data)`
-
-    If you supply an argument list, the function removes those arguments from the list that were
-    recognized by the Ice runtime.
+    Creates a new communicator.
 
     Parameters
     ----------
     args : list of str, optional
-        An argument list, such as sys.argv.
-    data : InitializationData, optional
-        An instance of InitializationData.
+        The command-line arguments. This function parses arguments starting with `--` and one of the
+        reserved prefixes (Ice, IceSSL etc.) as properties for the new communicator. If there is an argument starting
+        with `--Ice.Config`, this function loads the specified configuration file. When the same property is set in a
+        configuration file and through a command-line argument, the command-line setting takes precedence.
+    initData : InitializationData, optional
```

Comment:
I was undecided or whether keep `data` for this keyword parameter or update it to `initData`. I decided to changed it because previously `data` was used for both the initialization data and the config file.

Keeping data would make this backwards compatible:

```
Ice.initilize(data=initData)
```

Which now should be:

```
Ice.initialize(initData=initData)
```